pkg/runtime/runtime-manifest.go

```diff
@@ -376,7 +376,7 @@ func (m *RuntimeManifest) buildSourceData() (map[string]interface{}, error) {
 		typeOfT := val.Type()
 		for i := 0; i < val.NumField(); i++ {
 			f := val.Field(i)
-			name := strings.ToLower(typeOfT.Field(i).Name)
+			name := toLowerCase(typeOfT.Field(i).Name)
```

Comment:
Oh so this "jsonify's" the name. Previously we were making the entire name lower case which would mess up something like "MyThing" turning it into "mything" when it should be "myThing". Do I have that right?

Reply:
Yup, that's correct!